### DISCUSSION TITLE
感谢开发者，OCR终于修理好了

### DISCUSSION THREAD

**Discussion by andydznj** [shuax/MouseInc.Settings#364] [General]
再次感谢开发者~~~

**Comment by ZHAOHAHA-Y** [comment]
是可以用了，但是识别成功率下降了，识别之后会少字

**Reply by kevzcr** [reply]
测试了一下
- 要识别的文本内容如下：

> C:\Users\D\Desktop\3\新建文件夹\DwnlData

- 识别效果：

> C:(Users(D(Desktop(3)新建文件DwnlData

- 将文本放大到最大，我权当软件是个人，放大给它看，效果如下：

> C:Users\D\Desktop(3)新建文件\DwnlData   

多认出来了3个 `\`

_ 总结：`夹` 字都少了，没看出来`夹`有什么特别之处，但两遍都没认出来。
 `\` 识别的比较差。
因为是文本，不是扫描图片，清晰度绝对有保证，Ai应该有Bug，因为它不是看不清，看样子是不认识，呵呵。

**Reply by andydznj** [reply]
的确会少字，不管你放大多少倍，怀疑是OCR服务方故意少的

**Comment by gbsix** [comment]
刚刚更新到2.13.1，又失效了

**Comment by shuax** [comment]
不是加了设置嘛，可以换接口啊。

**Reply by gbsix** [reply]
> 不是加了设置嘛，可以换接口啊。

刚换了飞浆，可以了。 昨天还好好的，今天升级到2.13.1就不行了，我还以为升级的问题。
谢谢大佬！

**Reply by ZHAOHAHA-Y** [reply]
有道最好用，但是有的网络用不了。其他两个不好用，一个识别率低，一个识别延迟高🤣

**Reply by Overbye** [reply]
> 

设置接口在哪里，没找到

**Reply by xiaomaomao3** [reply]
飞浆的延迟太高了，还是一开始自有的好用，但是失效了QAQ

**Reply by xiaomaomao3** [reply]
> > 
> 
> 接口设置在哪里，没找到
打开设置，功能开关，更多设置

**Reply by andydznj** [reply]
> 有道最好用，但是有的网络用不了。其他两个不好用，一个识别率低，一个识别延迟高🤣

现在默认是不是有道？

**Comment by xiaomaomao3** [comment]
飞浆的延迟太高了，还是一开始自有的好用，但是失效了QAQ

**Comment by shuax** [comment]
自有的 ocr.shuax.com 好的啊

**Reply by gbsix** [reply]
win7电信网络用不了，目前只有飞浆能用。

**Reply by 08spider** [reply]
我也只有飞浆能用。用浏览器可以打开ocr.shuax.com这个网址，但是MouseInc上用自有接口就识别不来

**Comment by shuax** [comment]
我开了tls1.3，可能系统太老链接不了

**Reply by gbsix** [reply]
希望继续支持win7啊

**Reply by Seryta** [reply]
> 我开了tls1.3，可能系统太老链接不了

我在internet选项中开启了使用tls1.3，但是”自有“仍然不可用

**Reply by Seryta** [reply]
> 我开了tls1.3，可能系统太老链接不了

的确是这个问题，我这边也成功了，在Internet选项中启用tls1.3、ssl3.0，参照https://blog.csdn.net/ximenjianxue/article/details/125958000 修改了注册表，重启系统“自用”OCR就可以用了

**Comment by mclovin-2k** [comment]
奇怪哦，我这里只有飞浆可以用。
自有和有道都报“未知错误”。
我是Win10，和TLS1.3应该无关吧。
